<ui version="4.0" >
 <class>Configuration</class>
 <widget class="QDialog" name="Configuration" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Application database</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="database_lineEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="database_browse_button" >
       <property name="text" >
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="overwrite_checkBox" >
     <property name="text" >
      <string>Overwrite existing files when downloading</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="instructions_checkBox" >
     <property name="text" >
      <string>Show instructions on startup</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Stop processing batch on</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="haltOn_comboBox" >
       <item>
        <property name="text" >
         <string>No errors</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Download error</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Install error</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>All errors</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Applications with no silent installer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="silentFirst_comboBox" >
       <item>
        <property name="text" >
         <string>First</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Last</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="confirm_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirm_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Configuration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirm_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Configuration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
